theHarvester/__main__.py

```diff
@@ -69,6 +69,56 @@ def start():
     virtual = args.virtual_host
     word = args.domain  # type: str
 
+    def store(search_engine, source, process_param=None, store_host=False, store_emails=False, store_ip=False, store_people=False,
+              store_data=False, store_links=False, store_results=False):
+        search_engine.process() if process_param is None else search_engine.process(process_param)
+        db_stash = stash.stash_manager()
+
+        if store_host:
+            host_names = filter(search_engine.get_hostnames())
+            all_hosts.extend(host_names)
+            db_stash.store_all(word, all_hosts, 'host', source)
+        if store_emails:
+            email_list = filter(search_engine.get_emails())
+            db_stash.store_all(word, email_list, 'email', source)
+        if store_ip:
+            ips_list = search_engine.get_ips()
+            all_ip.extend(ips_list)
+            db_stash.store_all(word, ips, 'ip', source)
+        if store_data:
+            data = filter(crtsh_search.get_data())
+            all_hosts.extend(data)
+            db.store_all(word, all_hosts, 'host', source)
+        if store_results:
+            emails, hosts, urls = search_engine.get_results()
+            all_emails.extend(emails)
+            hosts = filter(hosts)
+            global trello_urls
```

Comment:
Is the use of the global keyword truly necessary?

Reply:
we need to declare the variable as `global` because we are assigning it and then accessing it [here](https://github.com/laramies/theHarvester/blob/master/theHarvester/__main__.py#L499)

if we change L97 to `trello_urls.extend(filter(urls)` then we can skip the `global` keyword

Reply:
I would prefer not using global unless absolute necessary, in this case doing will suffice.